cli/cmd/service/create.go

```diff
@@ -17,10 +23,43 @@ func CreateCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
-			res1, err := client.CreateService(cmd.Context(), &adminv1.CreateServiceRequest{
+			// Parse attributes if provided
+			var attrs map[string]string
+			if attributes != "" {
+				if err := json.Unmarshal([]byte(attributes), &attrs); err != nil {
+					return fmt.Errorf("failed to parse --attributes as JSON: %w", err)
+				}
+			}
+
+			req := &adminv1.CreateServiceRequest{
 				Name:             args[0],
 				OrganizationName: ch.Org,
-			})
+			}
+
+			if orgRole == "" && projectRole == "" {
+				return fmt.Errorf("either --org-role or --project-role must be specified")
+			}
+
+			// Set org role if provided
+			if orgRole != "" {
+				req.OrgRoleName = orgRole
+			}
+
+			// Set project role if provided
+			if projectRole != "" {
+				if projectName == "" {
+					return fmt.Errorf("project name is required when project role is set")
```

Comment:
Better to be clear about the flag that's missing:

```suggestion
					return fmt.Errorf("you must set --project when using --project-role")
```